weave/trace/refs.py

```diff
@@ -1,4 +1,5 @@
 import dataclasses
```

Comment:
Major changes in this file are to ensure that we quote the user-controlled fields, similar to refs_internal